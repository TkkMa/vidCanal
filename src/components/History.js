import React from 'react';
import HistoryListFilters from './HistoryListFilters';
import HistoryList from './HistoryList';


const History = (props)=> {
    return(
            <div>
                <HistoryListFilters />
                <HistoryList {...props} />
            </div>
        )
}

export default History;