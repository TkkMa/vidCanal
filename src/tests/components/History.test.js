import React from 'react';
import {shallow} from 'enzyme';
import History from '../../components/History';

test('should render history page correctly', ()=>{
    const wrapper = shallow(<History />);
    expect(wrapper).toMatchSnapshot();
})