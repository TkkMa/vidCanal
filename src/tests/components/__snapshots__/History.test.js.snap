// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render history page correctly 1`] = `
<div>
  <Connect(HistoryListFilters) />
  <Connect(HistoryList) />
</div>
`;
